functional_tests.go

```diff
@@ -2647,26 +3100,37 @@ func testRemovePartiallyUploadedV2() {
 	}()
 
 	objectName := bucketName + "-resumable"
+	args["objectName"] = objectName
+
 	_, err = c.PutObject(bucketName, objectName, reader, "application/octet-stream")
 	if err == nil {
-		logger().Fatal("Error: PutObject should fail.")
+		failureLog(function, args, startTime, "", "PutObject should fail", err).Fatal()
```

Comment:
No point of logging nill error message, err:
failureLog(function, args, startTime, "", "PutObject should fail"~~, err~~).Fatal()